generator/03-syndesis-ui.yml.mustache

```diff
@@ -76,11 +76,17 @@
           image: syndesis/syndesis-ui:latest
 {{/Dev}}
           imagePullPolicy: IfNotPresent
+          livenessProbe:
+            httpGet:
+              path: "/"
+              port: 2015
+            timeoutSeconds: 5
           readinessProbe:
             httpGet:
               path: "/"
               port: 2015
             initialDelaySeconds: 1
+            timeoutSeconds: 5
```

Comment:
Probably not, just wanted it to make symmetric (and if it's a systematic, non-spring boot issue, then it would cover this case, too). Also, it doesn't harm I think.